sdk/resources/Azure.ResourceManager.Resources/api/Azure.ResourceManager.Resources.net8.0.cs

```diff
@@ -1117,6 +1117,7 @@ protected virtual void JsonModelWriteCore(System.Text.Json.Utf8JsonWriter writer
     public partial class ArmDeploymentContent : System.ClientModel.Primitives.IJsonModel<Azure.ResourceManager.Resources.Models.ArmDeploymentContent>, System.ClientModel.Primitives.IPersistableModel<Azure.ResourceManager.Resources.Models.ArmDeploymentContent>
     {
         public ArmDeploymentContent(Azure.ResourceManager.Resources.Models.ArmDeploymentProperties properties) { }
+        public Azure.ResourceManager.Resources.Models.DeploymentIdentity Identity { get { throw null; } set { } }
```

Comment:
well I think the type here should be replaced by ManagedServiceIdentity from resourcemanager.